config/packages/doctrine.yaml

```diff
@@ -14,6 +14,7 @@ doctrine:
             information_complementaire: App\Entity\DoctrineType\Signalement\InformationComplementaireType
     orm:
         auto_generate_proxy_classes: true
+        #enable_lazy_ghost_objects: true
```

Comment:
ah oui ok, c'est ce dont tu parlais ce matin. 
Donc non, ne pas virer cette ligne, et voir si Saidi ou Emilien ont un avis

Reply:
https://github.com/doctrine/DoctrineBundle/issues/1651

D'après ce que j'ai lu c'est une nouvelle manière qu'a doctrine sur le chargement des objets dans les relations 
Il est en mode featture flag mais à un moment donné, il faudra la mettre à true
> This is an opt-in for the new way to manage proxies in the ORM (which requires symfony/var-exporter 6.2+).

Normalement ça devrait marcher (tu peux tenter clear cache en mode bourrin (`rm -rf var/cache/*`))
Si ça ne marche pas, poster l'erreur, décommenter en mettant à false on verra plus tard

> As the code should be working fine now, I see no reason to set it to false when you are on Symfony 6.2+, especially in new projects. Set it to false as a workaround for issues you might face when upgrading (so for an upgrade, try setting it to true and if you don't have bugs, leave it like that, otherwise report the issue and switch it back to false until the issue is fixed

Reply:
> [doctrine/DoctrineBundle#1651](https://github.com/doctrine/DoctrineBundle/issues/1651)
> 
> D'après ce que j'ai lu c'est une nouvelle manière qu'a doctrine sur le chargement des objets dans les relations Il est en mode featture flag mais à un moment donné, il faudra la mettre à true
> 
> > This is an opt-in for the new way to manage proxies in the ORM (which requires symfony/var-exporter 6.2+).
> 
> Normalement ça devrait marcher (tu peux tenter clear cache en mode bourrin (`rm -rf var/cache/*`)) Si ça ne marche pas, poster l'erreur, décommenter en mettant à false on verra plus tard
> 
> > As the code should be working fine now, I see no reason to set it to false when you are on Symfony 6.2+, especially in new projects. Set it to false as a workaround for issues you might face when upgrading (so for an upgrade, try setting it to true and if you don't have bugs, leave it like that, otherwise report the issue and switch it back to false until the issue is fixed

J'ai essayé plein de trucs dont le cache, je n'ai plus l'erreur de tête mais il y'en a une qui fait planter toutes les pages que j'avais réussi à contourner mais il en restait derrière d'autre en lien avec la sérialisation d'object Proxy de doctrine et la j'ai eu beau chercher je n'ai rien trouvé pour le régler